norsk-henvisningsstandard-for-rettsvitenskapelige-tekster.csl

```diff
@@ -0,0 +1,318 @@
+  <macro name="author-short">
+    <names variable="author">
+      <name form="short" et-al-min="4" et-al-use-first="3" delimiter-precedes-last="never" name-as-sort-order="first" and="text"/>
+      <et-al term="and others"/>
+      <substitute>
+        <names variable="editor"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-full">
+    <names variable="author">
+      <name et-al-min="4" et-al-use-first="3" delimiter-precedes-last="never" name-as-sort-order="first" and="text"/>
+      <et-al term="and others"/>
+      <substitute>
+        <names variable="editor"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="editor">
+    <names variable="editor">
+      <name/>
+    </names>
+  </macro>
+  <macro name="title-short">
+    <choose>
+      <if type="article-journal chapter" match="any">
+        <text quotes="true" variable="title"/>
+      </if>
+      <else-if type="legislation">
+        <text variable="title-short"/>
+      </else-if>
+      <else-if type="legal_case">
+        <text variable="title"/>
+      </else-if>
+      <else>
+        <text font-style="italic" variable="title-short"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="title">
+    <choose>
+      <if type="chapter article-journal article-newspaper article-magazine" match="any">
+        <text quotes="true" variable="title"/>
+      </if>
+      <else-if type="legal_case legislation" match="any">
+        <text variable="title"/>
+      </else-if>
+      <else>
+        <text font-style="italic" variable="title"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="issued">
+    <choose>
+      <if type="personal_communication" match="none">
+        <date prefix="(" suffix=")" variable="issued">
+          <date-part name="year"/>
+        </date>
+      </if>
+      <else-if type="personal_communication" match="all">
+        <date variable="accessed" prefix="(" suffix=")">
+          <date-part name="year"/>
+        </date>
+      </else-if>
+      <else>
+        <text term="no date" prefix="(" suffix=")"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="issued-full-date">
+    <date variable="issued">
+      <date-part name="day" form="numeric" suffix=". "/>
+      <date-part name="month" form="long" suffix=" "/>
+      <date-part name="year"/>
+    </date>
+  </macro>
+  <macro name="issued-no-parenthesis">
+    <choose>
+      <if variable="issued">
+        <date variable="issued">
+          <date-part name="year"/>
+        </date>
+      </if>
+      <else>
+        <text term="no date"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="retrieved-from">
+    <choose>
+      <if type="article-journal">
+        <text variable="archive" text-case="capitalize-first" prefix="(Sitert fra " suffix=")"/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="accessed-date">
+    <choose>
+      <if type="personal_communication">
+        <date variable="accessed" prefix=", e-post, ">
```

Comment:
e-post I assume means email? That's not ideal since "personal_communication" can also be, e.g., a letter. Could you use `genre` here?

Reply:
Yes. I have attempted to edit this to now use the "genre" CSL variable. I also changed the date to "issued" rather than "accessed", which makes more sense (at least to me).